pkg/resources/sop_url.go

```diff
@@ -12,6 +12,7 @@ const (
 	sopUrlRedisResourceStatusPhaseFailed     = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/redis_resource_status_phase_failed.asciidoc"
 	sopUrlPostgresWillFill                   = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/postgres_storage_alerts.asciidoc"
 	sopUrlRedisMemoryUsageHigh               = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/redis_memory_usage_high.asciidoc"
+	sopUrlPostgresMemoryUsageHigh            = "TODO"
```

Comment:
Should we add a link for this SOP?

Reply:
Guess this was pulled out to another pr. No Postgres Memory alerts.

Reply:
good catch, looks like there is no SOP added for the Alert added in this PR